src/pybasic/pybasic.py

```diff
@@ -125,6 +130,7 @@ class BaSiC(BaseModel):
     _reweight_score: float = PrivateAttr(None)
     _flatfield: np.ndarray = PrivateAttr(None)
     _darkfield: np.ndarray = PrivateAttr(None)
+    _logger: logging.Logger = PrivateAttr(None)
```

Comment:
Pydantic won't dump `_logger`, right?

Reply:
Did you figure anything out for this?

Reply:
I tried with the `pydantic` [test code](https://pydantic-docs.helpmanual.io/usage/models/#private-model-attributes) and it seems the members annotated by `PrivateAttr` does not appear either of `model.dict()` or `model.json()`!

Reply:
I still don't understand the need for additional complexity here. Creating a unique logger for each model seems like an unnecessary complication. Why not use the file level logger here?

Reply:
I thought it may be suitable in cases where we have multiple BaSiC objects and want to track the logs independently.

Reply:
> I don't think we need to have logger be part of the pydantic class. I think the code can just use the file-level logger.
Not sure why this was marked as resolved.

Hi @Nicholas-Schaub, sorry, it seems I missed your comment yesterday and marked it as resolved as I answered Tim's first comment. Assuming most of the users don't use the multiple objects simultaneously, maybe we can output the log in `basicpy.BaSiC` level.

Reply:
I just switched to the file-level logger.